app/src/debug/java/com/mapbox/mapboxsdk/plugins/testapp/activity/TestActivity.kt

```diff
@@ -7,47 +7,47 @@ import com.mapbox.mapboxsdk.plugins.testapp.R
 
 class TestActivity : AppCompatActivity() {
 
-  lateinit var mapView: MapView
-
-  override fun onCreate(savedInstanceState: Bundle?) {
-    super.onCreate(savedInstanceState)
-    setContentView(R.layout.activity_test)
-    this.mapView = findViewById(R.id.mapView)
-    mapView.onCreate(savedInstanceState)
-  }
-
-  override fun onStart() {
-    super.onStart()
-    mapView.onStart()
-  }
-
-  override fun onResume() {
-    super.onResume()
-    mapView.onResume()
-  }
-
-  override fun onPause() {
-    super.onPause()
-    mapView.onPause()
-  }
-
-  override fun onStop() {
-    super.onStop()
-    mapView.onStop()
-  }
-
-  override fun onSaveInstanceState(outState: Bundle) {
-    super.onSaveInstanceState(outState)
-    mapView.onSaveInstanceState(outState)
-  }
-
-  override fun onDestroy() {
-    super.onDestroy()
-    mapView.onDestroy()
-  }
-
-  override fun onLowMemory() {
-    super.onLowMemory()
-    mapView.onLowMemory()
-  }
+    lateinit var mapView: MapView
```

Comment:
our java codestyle guidelines indicate to use a tab size of 2 instead of 4. I feel we should do the same for kotlin. In gl-native, I'm enforcing this with the following configuration file https://github.com/mapbox/mapbox-gl-native/pull/15052/files#diff-fec7f0b28f89ca0bd4eae384988b2592

Reply:
Per https://github.com/jeremymailen/kotlinter-gradle#configuration, https://github.com/mapbox/mapbox-plugins-android/pull/1006/commits/ff6c3f31cf328922571e39afb1bb78c2901c1599 sets the indentSize to 2, but `make kotlin-lint` didn't prompt me to fix the `TestActivity.kt` spacing as seen above 😫  . There were other weird config errors too `:rabbit-hole:`. Anyways, I've got https://github.com/jlleitschuh/ktlint-gradle working much better. Going to close this pr and open a new one with the https://github.com/jlleitschuh/ktlint-gradle implementation.